app/views/podcast_planner/_form_side.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="card">
+  <div class="card-header-primary">
+    <h5 class="card-title">Schedule Preview</h5>
+  </div>
+  <%= turbo_frame_tag "preview", target: "_top" do %>
+    <% if @planner.dates.present? %>
+      <%= form_with(
+            url: podcast_planner_path(@podcast),
+            method: :put
+          ) do |form| %>
+          <div class="card-body">
+            <p><%= "Publish Time: #{l(@planner.publish_time, format: :time_12_hour)}" %></p>
```

Comment:
yeah, i'm wondering if i just put this and the `segment_count` inputs here? instead of updating them in the "main" partial.

Reply:
Once this merges, may be worth another pass on the UX.  I was also thinking the segment-count input should be a lot higher on the page, so they don't miss it.

(It would be a pain to generate dozens of episodes with the wrong segment_count, and have to go update them all one by one).